src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -4,6 +4,7 @@ import ChatSchema from '../../../database/schema/ChatSchema';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import UserSchema from '../../../database/schema/UserSchema';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import useSystemMessage from './useSystemMessage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useSystemMessage` is a good addition as it helps to separate the logic for handling system messages into its own hook, improving modularity and maintainability.
```diff
+ import useSystemMessage from './useSystemMessage';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->